docs/antrea-network-policy.md

```diff
@@ -1296,6 +1301,47 @@ Note: Antrea will use a reserved label key for internal processing `serviceAccou
 The reserved label looks like: `internal.antrea.io/service-account:[ServiceAccountName]`. Users should avoid using
 this label key in any entities no matter if a policy with `serviceAccount` is applied in the cluster.
 
+### ACNP appliedTo NodePort Service
```

Comment:
Probably name it "Apply to NodePort Service" or "appliedTo with NodePort Service" or "NodePort Service in appliedTo"

Reply:
Done.